molgenis-core-ui/src/main/java/org/molgenis/data/importer/wizard/OptionsWizardPage.java

```diff
@@ -92,7 +92,10 @@ public String handleRequest(HttpServletRequest request, BindingResult result, Wi
 					fileRepositoryCollectionFactory.createFileRepositoryCollection(tmpFile).getFileNameExtensions());
 
 			File file = new File(tmpFile.getParent(), userGivenName + "." + extension);
-			tmpFile.renameTo(file);
+			if (!tmpFile.renameTo(file))
+			{
+				LOG.error("Failed to rename '{}' to '{}'", tmpFile.getName(), file.getName());
```

Comment:
If you expect it can actually happen, add error to the result. Otherwise, don't just log but also throw exception.

Reply:
A failed rename attempt of a system temporary file doesn't have to impact the user import.

Reply:
If it fails, that probably is because it already exists and someone else has already imported a file with that name. I think the whole renaming is very hairy here. Will file a separate issue for these things.